src/sonic-config-engine/minigraph.py

```diff
@@ -257,7 +257,12 @@ def parse_dpg(dpg, hname):
         aclintfs = child.find(str(QName(ns, "AclInterfaces")))
         acls = {}
         for aclintf in aclintfs.findall(str(QName(ns, "AclInterface"))):
-            aclname = aclintf.find(str(QName(ns, "InAcl"))).text.upper().replace(" ", "_").replace("-", "_")
+            try:
+                aclname = aclintf.find(str(QName(ns, "InAcl"))).text.upper().replace(" ", "_").replace("-", "_")
+                stage = "ingress"
+            except:
+                aclname = aclintf.find(str(QName(ns, "OutAcl"))).text.upper().replace(" ", "_").replace("-", "_")
+                stage = "egress"
```

Comment:
True. However, the scenario you describe also existed previously if there was no `InAcl` element defined. It doesn't seem to have posed an issue before, but I agree that we should probably print an error message. Just wondering if we should skip the table or stop parsing the minigraph altogether and exit in error because it is not formatted properly if there is neither an `InAcl` or `OutAcl` element.

Reply:
I think in most places where there is information missing we skip the table, but since the current behavior is to throw an exception I think it would make sense to exit with an error.

Reply:
Addressed in latest commit.